interns-react-weekly-assignments/src/components/Home/Ritika/Week3/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CounterApp from './CounterComponent/Counter';
+import AutoFocus from './AutoFocus/AutoFocus';
+import FriendStatus from './FriendStatus/ShowFriendStatus/FriendStatus';
+import { Week3Wrapper } from './index.styled';
+
+const RWeek3 = () => {
```

Comment:
Give proper name to components of your React application. By looking at the name, one should be able to understand that which component it could be. (it can be "RitikaWeek3")

Reply:
It was suggested from our mentors not use name in the components ,that is why I avoided using it , I might change this to MainWeek3Wrapper may be.